### PR TITLE
Convert all issue templates into issue forms

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Convert all issue templates into issue forms.
  - Also disables blank issues and provides an `Other` issue template.

## How to test

- This would have to be merged to `master` to test it directly. Although, check [**this**](https://github.com/EddieHubCommunity/support) repository and try to submit an issue.

### CODE REVIEW

**Review by theblobscp — DISMISSED** [review]
Looks nice, although I have no idea how exactly this would work

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    attributes:
+      label: Problem
+      description: Tell us about the current problem that you face that this feature might help you with.
+    validations:
+      required: true
+  - type: textarea
+    id: suggestedfeat
+    attributes:
+      label: Suggested feature
+      description: Tell us about the current problem that you face that this feature might help you with.
```
Comment (Thomas-Boi) [inline .github/ISSUE_TEMPLATE/feature_request.yml:17]:
```suggestion
      description: Tell us about the feature that might solve the above problem.
```

File: .github/ISSUE_TEMPLATE/icon_request.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    attributes:
+      label: About the icon
+      description: Short description of why you think this icon belongs in our project.
+    validations:
+      required: true
+  - type: textarea
+    id: links
+    attributes:
+      label: Links and sources
+      description: Provide links to the icon's official website/repository. Anywhere that shows us what the technology is about and its official logo. If available, also provide some resources where the icon/SVG can be found (Font Awesome, Icomoon, etc.).
```
Comment (Thomas-Boi) [inline .github/ISSUE_TEMPLATE/icon_request.yml:17]:
```suggestion
      description: Provide links to the icon's official website/repository/anywhere that shows what the technology is about and its official logo. If available, also provide some resources where the icon/SVG can be found (Font Awesome, Icomoon, etc.).
```

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Overall, this looks great. Just need to fix a few sentences and we are good

**Review by Thomas-Boi — APPROVED** [review]
This is great! Can't wait to see it in action soon 👍

**Pull request merged**